src/pages/AdvancedOrders/index.tsx

```diff
@@ -34,7 +38,7 @@ export default function AdvancedOrdersPage() {
         </styledEl.PrimaryWrapper>
 
         <styledEl.SecondaryWrapper>
-          <OrdersTableWidget additionalOrders={emulatedTwapOrders} />
+          <OrdersTableWidget additionalOrders={allEmulatedOrders} />
```

Comment:
Now we pass TWAP virtual orders separately, we don't mix them in the common orders store